extensions/interactions/MathEquationInput/directives/oppia-interactive-math-equation-input.component.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the MathEquationInput interaction.
+ *
+ * IMPORTANT NOTE: The naming convention for customization args that are passed
+ * into the directive is: the name of the parameter, followed by 'With',
+ * followed by the name of the arg.
+ */
+
+require('services/guppy-configuration.service.ts');
+require('services/math-interactions.service.ts');
+require('services/guppy-initialization.service.ts');
+require(
+  'interactions/MathEquationInput/directives/' +
+  'math-equation-input-rules.service.ts');
+require(
+  'pages/exploration-player-page/services/current-interaction.service.ts');
```

Comment:
Sort these alphabetically, please. Ditto in other places.

Reply:
Done.